### ISSUE TITLE
Google maps embed 

### ISSUE THREAD

**Issue by rachelcsimmons-cda** [GoogleChrome/web.dev#10117]
The Google map embed causes an accessibility error because its navigation table does not have a header. 

![Screen Shot 2023-05-31 at 5 03 15 PM](https://github.com/GoogleChrome/web.dev/assets/86799785/d7f17903-6e38-4ca5-ad18-dd7d9913a1b4)

**Comment by rachelandrew** [comment]
@rachelcsimmons-cda which page of web.dev is this embed on or are you reporting a general Google maps issue? If the latter this isn't the right place as this is the repo for the website hosted at web.dev.
